Week-3/assignment-1/index.js

```diff
@@ -0,0 +1,41 @@
+function add(a = 0, b = 0) {
```

Comment:
@chinmay4o the assignment asks for accepting n numbers of args. This fn will only add 2